pkg/controllers/provisioning/suite_test.go

```diff
@@ -2583,6 +2584,599 @@ var _ = Describe("Provisioning", func() {
 			})
 		})
 	})
+
+	Context("MinValuesPolicy", func() {
```

Comment:
Why do we need tests for both zone and instance type? They shouldn't be fundamentally different, right? I feel like we could collapse this to only testing one and have the same coverage.